misc/python/materialize/parallel_workload/action.py

```diff
@@ -229,6 +230,9 @@ def errors_to_ignore(self, exe: Executor) -> list[str]:
         return result
 
     def run(self, exe: Executor) -> None:
+        if exe.db.cheap:
```

Comment:
This was initially not that clear to me, although I could guess what it might be. Some comment or a rename to something like `only_cheap_checks` or `exclude_expensive_checks` could be helpful.